forge/test/models/pytorch/text/qwen/test_qwen_version_two.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+import forge
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+# Variants for testing
+variants = [
+    "Qwen/Qwen2.5-0.5B",
+    "Qwen/Qwen2.5-0.5B-Instruct",
+    "Qwen/Qwen2.5-1.5B",
+    "Qwen/Qwen2.5-1.5B-Instruct",
+    "Qwen/Qwen2.5-3B",
+    "Qwen/Qwen2.5-3B-Instruct",
+    "Qwen/Qwen2.5-7B",
+    "Qwen/Qwen2.5-7B-Instruct",
+    "Qwen/Qwen2.5-14B",
+    "Qwen/Qwen2.5-14B-Instruct",
+    "Qwen/Qwen2.5-32B",
+    "Qwen/Qwen2.5-32B-Instruct",
+    "Qwen/Qwen2.5-72B",
+    "Qwen/Qwen2.5-72B-Instruct",
+]
+
+
+@pytest.mark.parametrize("variant", variants, ids=variants)
```

Comment:
Do all variants pass compile? If no, then we should have xfail as parameter of the test...

Reply:
Do we really need to test all the variants, I mean there are models with 72B params!

Reply:
Do we really need all variants (there are 72B models!!)?

Reply:
Yeah, I've removed some of the larger variants